index.md

```diff
@@ -40,6 +40,25 @@ is contained within parenthesis, e.g., `feat(parser): add ability to parse array
 Commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, `chore:`, but these tags are
 not mandated by the conventional commits specification.
 
+## Examples
+
+### Commit message with no body
+```
+fix: Correct spelling of CHANGLOG
```

Comment:
changing the CHANGELOG should be docs.

Reply:
Ironically, I got this example from this repo. https://github.com/conventional-changelog/conventionalcommits.org/pull/13

But I'll fix it.

Reply:
Well, it was wrong :)

Reply:
subject should not start with caps, this spells trouble 🗡